### PR TITLE
Add Indonesian Translate

### PR DESCRIPTION
**Pull request by barraIhsan**
Add Indonesian (`id_id.json`) Translate

### CODE REVIEW

File: src/main/resources/assets/mcg/lang/id_id.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "mcg.button.back": "Kembali",
+  "mcg.coordinate.creationtitle": "Buat Kordinat",
+  "mcg.coordinate.creationfail": "Gagal menulis kordinat.",
+  "mcg.coordinate.readfail": "Gagal memuat kordinat dari berkas.",
+  "mcg.coordinate.removefail": "Gagal menghapus kordinat dari berkas.",
+  "mcg.coordinate.coordinatesof": "Kordinat dari",
+  "mcg.coordinate.newcoordinate": "Buat/Edit Kordinat",
+  "mcg.coordinate.removecoordinate": "Hapus Kordinat",
+  "mcg.coordinate.setoverlay": "Overlay Coordinate ",
+  "mcg.coordinate.clearoverlay": "Clear Overlay ",
```
Comment (AMereBagatelle) [inline src/main/resources/assets/mcg/lang/id_id.json:23]: I think there are a few missed here.

Comment (barraIhsan) [reply src/main/resources/assets/mcg/lang/id_id.json:23]: Oh yeah. I will translate it

**Comment by AMereBagatelle** [comment]
Not sure what's going on with Github Actions, but it doesn't seem to be your fault
It looks good, I'll merge soon

**Comment by barraIhsan** [comment]
Yeah, Adding translation string should not mess up with the Gradle build

**Pull request merged**